card/spec/lib/card/cache_spec.rb

```diff
@@ -108,7 +108,7 @@
       it "should work" do
         @cache.write('(汉语漢語 Hànyǔ; 华语華語 Huáyǔ; 中文 Zhōngwén', "foo")
         @cache.write('русский', "foo")
-        cache3 = Card::Cache.new store: @store, prefix: "prefix"
+        cache3 = Card::Cache.new store: @hard, prefix: "prefix"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.